woocommerce/woocommerce-ios
# Storage schema update: add `buttonText` to `Product`

Storage layer change for #2200 

## Changes

- Created a new model version 28 based on 27, and set v28 to the current model version
- Added `buttonText: String` for external products that corresponds to the `button_text` field in the [API response](https://woocommerce.github.io/woocommerce-rest-api-docs/#product-properties) (for non-external products, this field is empty)

<img width="1145" alt="Screen Shot 2020-06-05 at 3 32 21 PM" src="https://user-images.githubusercontent.com/1945542/83849295-c8b15880-a741-11ea-82bb-cdd7a415dd9d.png">


## Testing

Other than CI, it'd be nice to test migration from the previous model version:

- Launch the app from `develop` in logged in state
- Launch the app from the PR branch and sanity check as everything should work as before

Update release notes:
